test/nbrowser/gristUtils.ts

```diff
@@ -255,6 +255,10 @@ export async function selectAll() {
   await driver.executeScript('document.activeElement.select()');
 }
 
+export async function getActiveElement() {
```

Comment:
I don't think this is needed, since webdriver's built-in `driver.switchTo().activeElement()` should give the same result. (It's a weirdly named combination, but just returns the active element.)

Reply:
Thanks!